src/IRrecv.cpp

```diff
@@ -427,17 +432,18 @@ bool IRrecv::matchAtLeast(uint32_t measured, uint32_t desired,
   DPRINT(". Matching: ");
   DPRINT(measured);
   DPRINT(" >= ");
-  DPRINT(ticksLow(std::min(desired, TIMEOUT_MS * 1000), tolerance));
+  DPRINT(ticksLow(std::min(desired, (uint32_t) MS_TO_USEC(irparams.timeout)),
+                  tolerance));
   DPRINT(" [min(");
   DPRINT(ticksLow(desired, tolerance));
   DPRINT(", ");
-  DPRINT(ticksLow(TIMEOUT_MS * 1000, tolerance));
+  DPRINT(ticksLow(MS_TO_USEC(irparams.timeout), tolerance));
   DPRINTLN(")]");
   // We really should never get a value of 0, except as the last value
   // in the buffer. If that is the case, then assume infinity and return true.
   if (measured == 0) return true;
-  return measured >= ticksLow(std::min(desired, TIMEOUT_MS * 1000),
-                              tolerance);
+  return measured >= ticksLow(
+      std::min(desired, (uint32_t) MS_TO_USEC(irparams.timeout)), tolerance);
```

Comment:
No. Lint and compiler error/warnings otherwise. The later parameter defaults to an int.